modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ManagementToolbar.tsx

```diff
@@ -12,38 +12,34 @@
  * details.
  */
 
-import ClayButton from '@clayui/button';
-import ClayIcon from '@clayui/icon';
-import ClayManagementToolbar from '@clayui/management-toolbar';
-import {useModal} from '@clayui/modal';
 import {
 	API,
-	ModalEditExternalReferenceCode,
+	ManagementToolbar as GenericManagementToolbar,
 } from '@liferay/object-js-components-web';
-import classNames from 'classnames';
-import {navigate} from 'frontend-js-web';
-import React, {useState} from 'react';
+import React from 'react';
 
-export default function ManagementToolbar({
-	backURL,
-	externalReferenceCode: initialExternalReferenceCode,
-	hasPublishObjectPermission,
-	hasUpdateObjectDefinitionPermission,
-	isApproved,
-	label,
-	objectDefinitionId,
-	portletNamespace,
-	screenNavigationCategoryKey,
-	system,
-}: IProps) {
-	const [externalReferenceCode, setExternalReferenceCode] = useState<string>(
-		initialExternalReferenceCode
-	);
-	const [visibleModal, setVisibleModal] = useState<boolean>(false);
+interface IProps {
```

Comment:
@naraandrade, same as above, we can rename this interface to making it more specific, if you decided to change the name this whole ocmponent we can use `ObjectManagementToolbarProps`.